### DISCUSSION TITLE
Bootstrap Modal not clickable, backdrop occupies the whole screen

### DISCUSSION THREAD

**Discussion by pavankjadda** [twbs/bootstrap#30874] [General]
Bootstrap Modal not clickable, backdrop occupies the whole screen as shown in image. The issue happens in Chrome, Firefox and Safari

I am using 
"bootstrap": "^4.5.0",
"popper.js": "^1.16.1",

<img width="1248" alt="Screen Shot 2020-05-20 at 5 59 23 PM" src="https://user-images.githubusercontent.com/17564080/82501861-f5b80700-9ac3-11ea-8213-0bb35bcc9ca1.png">

**Comment by mdo** [comment]
_You should always include a **live demo** of the problem. Create a reduced test case via [CodePen](https://codepen.io/) or [JS Bin](https://jsbin.com/) and report back with your link, Bootstrap version, and specific browser and operating system details._
